src/encoded/static/components/cart/remove_multiple.js

```diff
@@ -116,19 +123,22 @@ const CartRemoveElementsComponent = ({
     };
 
     return (
-        <>
+        <div className="remove-multiple-control">
             <button
                 type="button"
-                disabled={loading || inProgress || (savedCartObj && savedCartObj.locked)}
+                disabled={nonSeriesDatasets.length === 0 || loading || inProgress || (savedCartObj && savedCartObj.locked)}
                 className="btn btn-info btn-sm"
                 onClick={handleOpenClick}
             >
                 Remove selected items from cart
             </button>
+            <div className="remove-multiple-control__note">
+                Any datasets belonging to a series remain after removing the selected items from the cart.
+            </div>
             {isModalVisible
```

Comment:
Maybe use the ??-operator ..?

Reply:
`isModalVisible` is a boolean, so I can use &&.